story/ep1/pt/umi1_6.txt

```diff
@@ -69,7 +69,7 @@
 ` Oho?!`
 ` Parece que as coisas estão começando a se encaixar, não acha?!"`
 `"Hahahahaha!`
-` Bem, nós descobrimos muita coisa quando tentamos resolvê‐lo há dois anos atrás.`
+` Bem, nós descobrimos muita coisa quando tentamos resolvê‐lo há dois anos.`
```

Comment:
why you hate "atrás"?

Reply:
Não precisa do "atrás" quando se usa o "há". Mas eu posso reverter

Reply:
deixe assim.